include/libopencm3/stm32/f0/adc.h

```diff
@@ -52,17 +52,16 @@
 /* Register definitions                                                      */
 /*****************************************************************************/
 
+#define ADC_SMPR(adc)			ADC_SMPR1(adc)
+#define ADC_TR(adc)			ADC_TR1(adc)
+
 #define ADC1_ISR			ADC_ISR(ADC)
 #define ADC1_IER			ADC_IER(ADC)
 #define ADC1_CR				ADC_CR(ADC)
 #define ADC1_CFGR1			ADC_CFGR1(ADC)
 #define ADC1_CFGR2			ADC_CFGR2(ADC)
-#define ADC1_SMPR1			ADC_SMPR1(ADC)
-#define ADC_SMPR(adc)			ADC_SMPR1(adc)  /* Compatibility */
-#define ADC1_SMPR			ADC_SMPR1(ADC) /* Compatibility */
-#define ADC1_TR1			ADC_TR1(ADC)
-#define ADC_TR(adc)			ADC_TR1(adc) /* Compatibility */
-#define ADC1_TR				ADC1_TR(ADC) /* Compatibility */
+#define ADC1_SMPR			ADC_SMPR(ADC)
+#define ADC1_TR				ADC_TR(ADC)
 #define ADC1_CHSELR			ADC_CHSELR(ADC)
 #define ADC1_DR				ADC_DR(ADC)
 #define ADC1_CCR			ADC_CCR(ADC)
```

Comment:
I  think @karlp wanted these removed as well, but you did not introduced them, even made the list smaller.
Maybe for another PR some day I guess...